haiku-apps/kottan/kottan-0.10.8.recipe

```diff
@@ -0,0 +1,43 @@
+SUMMARY="A viewer and editor for flattened BMessages"
+DESCRIPTION="Kottan lets you view and edit the contents of files that contain flattened BMessages"
+HOMEPAGE="https://github.com/andimachovec/Kottan"
+COPYRIGHT="2021 Andi Machovec"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/andimachovec/Kottan/archive/$portVersion-beta.tar.gz"
+CHECKSUM_SHA256="2b6e290838bf5a4015e9bdcaec53b967a0d0324d893ee17146f360b0f979c35d"
+SOURCE_DIR="Kottan-$portVersion-beta"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	kottan$secondaryArchSuffix = $portVersion
+	app:Kottan = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+	
+BUILD_PREREQUIRES="
```

Comment:
Trailing whitespace on the newline, this should be deleted (no newline between BUILD_REQUIRES and BUILD_PREREQUIRES):
```diff
	"
-
BUILD_PREREQUIRES="
```